backend/app/src/main/kotlin/io/tolgee/configuration/FiltersConfiguration.kt

```diff
@@ -0,0 +1,25 @@
+package io.tolgee.configuration
+
+import io.tolgee.component.ExceptionHandlerFilter
+import io.tolgee.component.VersionFilter
+import org.springframework.boot.autoconfigure.security.SecurityProperties
+import org.springframework.boot.web.servlet.FilterRegistrationBean
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class FiltersConfiguration {
```

Comment:
In Spring, anything that extends `Filter` is automatically registered as a filter. This is why there's a config class in the security package disabling registration of all filters.

This class disables the exception handler filter of the security chain (since it's part of the security chain), and configures the VersionFilter to run before the SecurityFilterChain.

The SecurityChain defined in WSC is the SecurityFilterChain; which is a single filter for the Servlet. The VersionFilter is outside of the security chain, and behaves as an independent servlet filter that Spring will autoregister.

This configures the order to make sure the version header is always applied.

Reply:
This also means that currently, all of the Tolgee filters are most likely ran twice. This PR fixes this which should be a performance relief.